xkcd/info.0.json

```diff
@@ -0,0 +1 @@
+{"month": "4", "num": 571, "link": "", "year": "2009", "news": "", "safe_title": "Can't Sleep", "transcript": "[[Someone is in bed, presumably trying to sleep. The top of each panel is a thought bubble showing sheep leaping over a fence.]]\n1 ... 2 ...\n<<baaa>>\n[[Two sheep are jumping from left to right.]]\n\n... 1,306 ... 1,307 ...\n<<baaa>>\n[[Two sheep are jumping from left to right. The would-be sleeper is holding his pillow.]]\n\n... 32,767 ... -32,768 ...\n<<baaa>> <<baaa>> <<baaa>> <<baaa>> <<baaa>>\n[[A whole flock of sheep is jumping over the fence from right to left. The would-be sleeper is sitting up.]]\nSleeper: ?\n\n... -32,767 ... -32,766 ...\n<<baaa>>\n[[Two sheep are jumping from left to right. The would-be sleeper is holding his pillow over his head.]]\n\n{{Title text: If androids someday DO dream of electric sheep, don't forget to declare sheepCount as a long int.}}", "alt": "If androids someday DO dream of electric sheep, don't forget to declare sheepCount as a long int.", "img": "http:\/\/imgs.xkcd.com\/comics\/cant_sleep.png", "title": "Can't Sleep", "day": "20"}
```

Comment:
Move all these json fixtures to a ./testdata directory.

```
xkcd/
xkcd/main.go
xkcd/testdata/
xkcd/testdata/info.0.json
xkcd/testdata/xkcd-index.json
...
```

The testdata directory is treated specially by go, it's contents are understood to not be intended for compilation. If for instance you are writing a tool that generates go code from other go code, this is quite useful as you can test it by putting testXXX.go files in ./testdata/ and they will be ignored by the go tool when you build your tool, but be available in your tests.